### ISSUE TITLE
Rollout Faction 1 or 1.0 gives rolloutFractionOutOfBound Exception

### ISSUE THREAD

**Issue by sachinvasista** [Microsoft/google-play-vsts-extension#103]
Rollout Faction 1 or 1.0 gives rolloutFractionOutOfBound Exception:

Please fix this.

Thanks in advance.

**Comment by lkillgore** [comment]
Hi @sachinvasista, could you please run with `system.debug = true` and reply with the results?

**Comment by lkillgore** [comment]
Hi @sachinvasista, I looked at this error message more closely.  It appears you're trying to use 'Google Play - Increase Rollout' or perhaps 'Google Play Promote' - Destination Track: 'Rollout' to do your release.  What you want to use is 'Google Play Promote' - Destination Track: 'Production' to release to everyone.

**Comment by sachinvasista** [comment]
Alright I got it now! Thanks. Rollout 100% means Production track.

**Comment by lkillgore** [comment]
Hi @sachinvasista, I think I know what the problem is, and I think I know of a work-around:
1) Go to your GooglePlayPromote task
2) Temporarily change your Destination Track to 'Rollout'
3) If the 'Rollout Fraction' written is '1.0', please change it to '0.0'
![image](https://user-images.githubusercontent.com/26548919/43076566-98584e6e-8e52-11e8-9b7c-8bd11de71d50.png)

4) Save your definition
5) Change Destination Track back to 'Production'
6) Save your definition

There's a bug in the task where the old value is retained if you change the Destination Track, and if it exists, we send it, and if the value sent is '1.0', then Google's APIs send back that failure.

Please let me know whether this works or not.  Our next bug fix release should fix this issue if this does fix your build definition.

**Comment by sachinvasista** [comment]
Hello, 
I followed your steps for work-around, but no luck.
1. Rollout Fraction was not written as 1.0, and infct it was empty field.
2. I saved the definition first by putting 0.0 and second my chnaging the dest. trcak to prod, I got the same error:
```
2018-07-26T06:43:32.2221229Z   code: 403,
2018-07-26T06:43:32.2221538Z   errors: 
2018-07-26T06:43:32.2222046Z    [ { domain: 'androidpublisher',
2018-07-26T06:43:32.2222382Z        reason: 'rolloutFractionOutOfRange',
2018-07-26T06:43:32.2222610Z        message: 'Rollout user fraction must be greater than 0.0 and less than 1.0' } ] }
```

**Comment by lkillgore** [comment]
Could you please send me a log with debugging turned on?

**Comment by lkillgore** [comment]
This should be fixed in the latest release: 2.141.0.  Please let us know if there are any further issues.  Thanks!
